excel/pom.xml

```diff
@@ -49,6 +49,36 @@
         <groupId>org.apache.maven.plugins</groupId>
         <artifactId>maven-compiler-plugin</artifactId>
       </plugin>
+      
+       <plugin>
+           <groupId>org.apache.maven.plugins</groupId>
+           <artifactId>maven-shade-plugin</artifactId>
+           <version>3.1.0</version>
+           <configuration>
+               <shadedArtifactAttached>true</shadedArtifactAttached>
+               <shadedClassifierName>shaded</shadedClassifierName>
+               <artifactSet>
+                   <includes>
```

Comment:
Is it necessary to have this? I thought `provided` dependencies would be excluded automatically. Or is there something else being done here?